### PR TITLE
fix: code refactoring:

### PR DESCRIPTION
**Pull request by SurprisedBird**
- rename "game" to "city"
- remove unnecessary import from effects files

### CODE REVIEW

File: city.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 @dataclass
-class Game:
+class City:
     citizens: List['Citizen'] = field(default_factory=list)
```
Comment (elias-gvin) [inline city.py:9]: Please move action_manager to game_controller. It should not be a Part of the city

File: city.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 
 @dataclass
-class Game:
+class City:
```
Comment (elias-gvin) [inline city.py:8]: Please, make base class for City and Citizen.
It should contain fields like:
context,
name,
effects

**Review by elias-gvin — CHANGES_REQUESTED** [review]

File: tests/unit/database_test.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 from citizens.citizen import Citizen
 from effects.database_effect import DataBase
 from effects.none_effect import NoneEffect
-from game import Game
+from game import City
 from game_controller import GameController
```
Comment (elias-gvin) [inline tests/unit/database_test.py:14]: please, rename file game.py to city.py

**Review by elias-gvin — CHANGES_REQUESTED** [review]

File: city.py (hunk 1 above)
Comment (asternotus) [reply city.py:9]: Done

File: tests/unit/database_test.py (hunk 3 above)
Comment (asternotus) [reply tests/unit/database_test.py:14]: Done

File: city.py (hunk 2 above)
Comment (asternotus) [reply city.py:8]: May be implement this later, not a critical

File: game_controller.py [hunk 4]
```diff
@@ -38,7 +39,7 @@ def start_game(self) -> None:
 # =================================================================
 
     def _prepare_game(self) -> None:
-        self._game.action_manager = ActionManager()
+        self._action_manager = ActionManager()
```
Comment (elias-gvin) [inline game_controller.py]: You've already initialized Action Manager in __init__. Please, remove this line

**Review by elias-gvin — APPROVED** [review]

**Review by elias-gvin — APPROVED** [review]

**Pull request merged**